front/app/components/UI/UserSelect/index.tsx

```diff
@@ -49,19 +50,14 @@ const UserSelect = ({
   className,
   id,
   inputId,
-  hideAvatar = false,
 }: DataProps & Props) => {
   const canLoadMore = users.lastPage !== users.currentPage;
-  const usersList: IUserData[] = Array.isArray(users.usersList)
-    ? users.usersList
-    : [];
-  const selectedUser = selectedUserId
-    ? usersList.find((user) => user.id === selectedUserId)
-    : null;
+  const usersList = Array.isArray(users.usersList) ? users.usersList : [];
```

Comment:
usersList was only 5 users (see props of GetUsers), so our selected user was not found (consistently) with the usersList.find (on the left).